ci/deploy/principal.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
If you're only deploying (and not building) you don't need this (see ci/prod or ci/integration)